README.md

```diff
@@ -310,6 +310,10 @@ The following snippet shows a typical Graphite configuration:
 | `graphite.poll-time`                                  | No       | Amount of time between Graphite polls. Defaults to `5000`. |
 | `graphite.poll-time-unit`                             | No       | Time unit of `graphite.poll-time` - this is [the list of allowed values](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/TimeUnit.html). Defaults to `MILLISECONDS`. |
 
+Prometheus can also be used to gather metrics. This can be done by enabling the Prometheus endpoint in the configuration:
```

Comment:
can you add there what the default "url" would be will just make it a little easier to find I assume it's normally printed startup logs?

Reply:
I think the default url is just `/prometheus`

Reply:
Done

Reply:
ah ok that trivial, excellent :)